Tests/test_AlignInfo.py

```diff
@@ -139,7 +139,7 @@ def test_pseudo_count(self):
         ic = summary.information_content(e_freq_table=expected,
                                          log_base=math.exp(1),
                                          pseudo_count=1)
-        ic_vector = summary.ic_vector.values()
+        ic_vector = list(summary.ic_vector.values())
```

Comment:
summary.ic_vector is a dict (not sure why...) and in py3, the return values of dict.values() is a view, so I couldn't access the ith element. 
But yes, you're right, on reason why zip was better. 
Also I add changes to guarantee the list order.